db/default/users.rb

```diff
@@ -56,7 +56,7 @@ def create_admin_user
     say "\nWARNING: There is already a user with the email: #{email}, so no account changes were made.  If you wish to create an additional admin user, please run rake spree_auth:admin:create again with a different email.\n\n"
   else
     admin = Spree::User.new(attributes)
-    admin.skip_confirmation!
+    admin.skip_confirmation!.skip_confirmation_notification!
```

Comment:
Interesting, I probably didn't see this one because I didn't have delayed job running locally.

Reply:
This one only occurs on a fresh deploy :point_up: